### ISSUE TITLE
tests with dev release

### ISSUE THREAD

**Issue by Ognian** [Spiritdude/OpenJSCAD.org#240]
Hello,
I've loaded your great lib with
 ```
"openjscad": "https://github.com/Spiritdude/OpenJSCAD.org/tarball/dev"
```
in the package.json dependencies section.
 `npm install`works. Very simple `test1.jscad`:
```
function main() {
    var cube = CSG.cube();
    return cube;
}
```
works via `node_modules/.bin/openjscad -v test1.jscad` also.

BUT  then I tried to
```
include("node_modules/jscad-utils/jscad-utils.jscad");
```
into the `test1.jscad` file and use `util` which is defined in the included file
I've got an error that `util` is not defined

I tried a lot of things but somehow it looks like include doesn't work...
Any ideas on this?
Thanks a lot
Ognian

**Comment by kaosat-dev** [comment]
Hi @Ognian !
Thanks for the feedback !
I'll try this out as well, could you please just confirm that jscad-utils is this one :
https://www.npmjs.com/package/jscad-utils

**Comment by Ognian** [comment]
@kaosat-dev yes it is this one.

**Comment by kaosat-dev** [comment]
@Ognian  ok I can reproduce I will see how hard it is to fix

**Comment by kaosat-dev** [comment]
@Ognian I fixed it in #241 and added some tests, I am still doing some cleanup work, but it will be merged soon

**Comment by Ognian** [comment]
WOW thanks a lot!
2 additional suggestions I had when I was looking into the project (just in case you didn't have the same ideas too):
- for the cli you could use [commander](https://github.com/tj/commander.js/)
- publish it on npm, even as alpha, so that the name is reserved

**Comment by kaosat-dev** [comment]
Hi @Ognian thanks for the feedback !
- I agree the CLI is very barebones right now, so `comander`would make sense (also it has only one dependency , always good in my book !)
- most of the sub packages are already published on NPM under the [jscad namespace](https://www.npmjs.com/org/jscad), so no risk of name squatting :) 

Btw I also found an additional issue with relative paths when using the CLI, so I added one more test and fixed it in #244

**Comment by Ognian** [comment]
tested with #244 in place --> IT WORKS.
Thanks a lot.
